docs/tutorial.md

```diff
@@ -878,6 +880,51 @@ local function do_something(p: MyPoint)
    -- ...
 end
 ```
+## Integers and numbers
+
+The integer sub-type was added with release 0.13.0. It is of undefined
+precision, deferring to the Lua VM.
+
+* integer constants without a dot (e.g. 123) are of type integer
```

Comment:
Stylistically, it would be better if this section was changed from bullet points into actual paragraphs, to match the rest of the tutorial.

Keep in mind that the tutorial is not a reference manual, so it aims to _teach_ a newcomer in a logical sequence, and just not be a sequential dump of info as in the release notes, neither a complete specification.

Perhaps some of this is too detailed for a tutorial, even. How about a short paragraph near the beginning in the "Types in Teal" section just covering the basics, and then a separate file in the docs/ folder specifically for the details about integers?